pkgs/by-name/tr/treedome/package.nix

```diff
@@ -22,12 +22,12 @@
 
 let
   pname = "treedome";
-  version = "0.4.5";
+  version = "0.5.1";
 
   src = fetchgit {
```

Comment:
Codeberg is a gitea-like instance, and we have a dedicated fetcher for it.

Reply:
done

Reply:
welp, gotta use the regular fetchgit, because the gitea fetch doesn't support lfs yet :(